Content.Shared/Ghost/SharedGhostComponent.cs

```diff
@@ -52,18 +79,21 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
     public class GhostComponentState : ComponentState
     {
         public bool CanReturnToBody { get; }
+        public bool CanGhostInteract { get; }
 
         public HashSet<string>? LocationWarps { get; }
 
         public Dictionary<EntityUid, string>? PlayerWarps { get; }
 
         public GhostComponentState(
             bool canReturnToBody,
+            bool canGhostInteract,
             HashSet<string>? locationWarps = null,
             Dictionary<EntityUid, string>? playerWarps = null)
```

Comment:
These need removing as they were jiggled around with the refactor (this is a reminder for futuresloth).